tests/integration/golang/aim/project/get_project_activity_test.go

```diff
@@ -0,0 +1,78 @@
+//go:build integration
+
+package run
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/response"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetProjectActivityTestSuite struct {
+	suite.Suite
+	client             *helpers.HttpClient
+	projectFixtures    *fixtures.ProjectFixtures
+	experimentFixtures *fixtures.ExperimentFixtures
+	runFixtures        *fixtures.RunFixtures
+	runs               []*models.Run
+	activity           response.ProjectActivity
```

Comment:
what is the point to store this entities?

Reply:
unsure, removing